client/gutenboarding/store/selectors.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Internal dependencies
+ */
+import { State } from './reducer';
+
+export const siteType = ( state: State ) => state.siteType;
```

Comment:
Not `getSiteType()`? 
```suggestion
export const getSiteType = ( state: State ) => state.siteType;
```

Or are we seeing the typed state :unicorn: where selectors become an afterthought and are really just getters? (In that case, do we still need them? Or will it become accepted to just use `state.something` in a `connect`/`withSelect`?)

Reply:
I'll use `getSiteType`, although that's a good point about typed state. We could start exploring a generic, top-level selector `getState = identity`, then use property access for the bits of state we're interested in.

For now, I'll leave this PR as "standard" as possible 👍